static/fetch.js

```diff
@@ -1,50 +1,39 @@
 (function(){
-	const RELOAD_TIME = 5000;
-	setInterval(loop, RELOAD_TIME);
+    const RELOAD_TIME = 3000;
+    let last_fetch = Date.now();
+    setInterval(() => {
+        loop(last_fetch).then(() => {
+            last_fetch = Date.now();
+        });
+    }, RELOAD_TIME);
 })();
 
-async function loop(){
-	let data = await reload();
-	for (let printer_name in data){
-		queue(printer_name, data[printer_name]);
-	}
+async function loop(last_fetch){
+    let data = await (await fetch(`/reload/recent?last-fetch=${last_fetch}`)).json();
+    for (let printer_name in data){
+        queue(printer_name, data[printer_name]);
+    }
 }
 
 function element(username, date){
-	let wrapper = document.createElement('div');
-	wrapper.classList.add('printlist-user');
-	let name = document.createElement('div');
-	name.classList.add('printlist-user-handle');
-	name.innerText = username.replace(/\</g, '&lt;').replace(/\>/g, '&gt;');
-	wrapper.appendChild(name);
-	setTimeout(()=>wrapper.remove(), 180000)
-	return wrapper;
+    let wrapper = document.createElement('div');
+    wrapper.classList.add('printlist-user');
+    let name = document.createElement('div');
+    name.classList.add('printlist-user-handle');
+    name.innerText = username;
+    wrapper.appendChild(name);
+    setTimeout(()=>wrapper.remove(), 180000)
```

Comment:
hardcode this in a const

Reply:
Can you clarify what you mean here?

Reply:
In general magic constants (180000) impact clarity. You moved it out in the latest commit so that works.